validering-jwks/jwks-test.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Hadde det gitt mening med en fil per organisasjon?

Reply:
Hvis vi skulle hel-automatisert det ville jeg sett for meg at hver organisasjon hadde stått for hvert sitt endepunkt.

Men jeg ser heller ikke for meg at det blir så mange nøkler mens vi holder på denne løsningen, vi kan starte med en fil 👍